src/NftDetails/NFTDetailsImage.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import {storiesOf} from '@storybook/react-native'
+import React from 'react'
+import {QueryClient, QueryClientProvider} from 'react-query'
+
+import {mocks, RouteProvider} from '../../storybook'
+import {SelectedWalletProvider} from '../SelectedWallet'
+import {NftDetailsImage} from './NftDetailsImage'
+
+storiesOf('NFT Details Image', module).add('Initial', () => {
```

Comment:
where is "loading" and "error"?

Reply:
I don't think there's a case where a user would be on the NFT Details Image screen and the NFT is not loaded or the NFT errored, as they need the NFT to be loaded first to open the details. However, I can add a "loading" and "error" state into stories, but there are no designs for such a state. Is returning a null okay for now?

Reply:
ah, then call it "default" instead of be "initial"